flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+  "automerge-flathubbot-prs": true,
+  "only-arches": ["x86_64"]
```

Comment:
this is disappointing given that the license is FLOSS.

Reply:
Hi, upstream contributor. Using prebuilt debian packages would give you armhf and aarch64 too.

Same building them from source. All targets are intended to be built on a x86_64 host with standard gcc cross toolchains.

Reply:
building from source is prefered. Also flathub build native. not cross.

Reply:
just to clarify aarch64 only is supported beide x86_64.

Reply:
Native builds won't work without minimal patching. If OP is interested in that route we would be able to make it work.

Reply:
> I managed to utilize the debs: [2a7d3021869b821b85bbad04bdf8b0caba1a17a7](https://github.com/flathub/flathub/pull/3724/commits/2a7d3021869b821b85bbad04bdf8b0caba1a17a7). Arm64 will be supported

Keep in mind the debian version isn't self contained as we try to repurpose host deps when it is possible: https://github.com/koreader/koreader/blob/master/platform/debian/do_debian_package.sh#L51

So fonts will be missing, unless you add them yourself to your build.

To be honest it would be easier for us to include a new target that does the same as the debian build without calling https://github.com/koreader/koreader/blob/master/platform/debian/do_debian_package.sh and then you should be able to build from source using a couple of commands.